pkgs/applications/blockchains/polkadot/default.nix

```diff
@@ -1,37 +1,43 @@
-{ stdenv
+{ clang
 , fetchFromGitHub
+, llvmPackages
+, protobuf
 , rustPlatform
-, pkgconfig
-, openssl
+, stdenv
```

Comment:
preference to eliminate unnecessary stdenv usage
```suggestion
, lib
```

Reply:
Updated and rebased.